index.bs

```diff
@@ -1246,25 +1246,35 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     is not capable of [=biometric recognition=], nor does it involve the presentation of a shared secret such as a password or
     PIN.
 
+: <dfn>User Account</dfn>
+:: In the context of this specification, a [=user account=] is some section of a [=[RP]=]'s services,
+    identified by a [=user handle=]
+    and associated with some set of [=credentials=] that a user may use to gain access to that section of services.
+    The set of credentials might change over time.
+    One user account might be accessed by one or more users and one user might have access to one or more user accounts,
+    depending on the user(s) and the [=[RP]=].
+
```

Comment:
```suggestion
    That is, a given [=user account=] might be accessed by one or more natural persons (also known as "users") and one natural person, or "user", might have access to one or more [=user accounts=],
    depending on actions of the user(s) and the [=[RP]=].

```

Reply:
I tweaked this a bit to make it flow a bit better. Thanks!